setup.py

```diff
@@ -52,7 +52,7 @@ def run(self):
 setup(
     name="pocketlint",
     description="Pocket-lint a composite linter and style checker.",
-    version="1.4.4",
+    version="1.4.4.c13",
```

Comment:
I added .c13 from your comment :) 
https://github.com/chevah/scame/pull/2#issuecomment-299626108

----

I'll look into moving this into a newer fork / and merging into the correct branch